subworkflows/local/phasing.nf

```diff
@@ -92,7 +91,7 @@ workflow PHASING {
                 .groupTuple()
                 .set { ch_hiphase_vcf }
 
-            TABIX_TABIX.out.tbi
+            TABIX_TABIX.out.csi
```

Comment:
Why did you change from tbi to csi?

Reply:
I changed it because TABIX_TABIX outputs a csi file, but I should really change TABIX_TABIX to output a tbi file. Will fix, thanks!

Reply:
Sounds good! You only need to add `--write-index=tbi` as an argument

Reply:
I take it back, TABIX_TABIX _should_ output a csi file. 
Since this workflow and phaser is not tested I didn't catch that `TABIX_TABIX.out.tbi` was empty and that `TABIX_TABIX.out.csi` _should_ have been used, as csi seems to be the default index for bcf. Will work on adding nf-test to this subworkflow to avoid this in the future (#273).